### DISCUSSION TITLE
`cell_methods` update

### DISCUSSION THREAD

**Discussion by BorjaEst** [xarray-contrib/cf-xarray#314] [Ideas]
Based on my question at https://github.com/cf-convention/discuss/issues/146
It seems it is recommended to update `cell_methods` after some operations (such mean, maximum, etc.).

**Comment by dcherian** [comment]
:+1: see #228, #253, #259. Right now we need some upstream work in xarray to make it happen: https://github.com/pydata/xarray/pull/5668
